configs/customisationsType.d.ts

```diff
@@ -12,6 +13,7 @@ export default interface customisations {
   },
   Staff_Cant_Punish_Staff_Restriction: {
     verify?: boolean,
+    add_note?: boolean,
```

Comment:
we should consider allowing staff to add staff notes for other staff members. imo it's not a "bad" thing like many other punishments